capture-sdk/screen-api-example-app/src/main/java/net/gini/android/capture/screen/ui/ConfigurationActivity.kt

```diff
@@ -0,0 +1,122 @@
+package net.gini.android.capture.screen.ui
+
+import android.app.Activity
+import android.content.Intent
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+import androidx.lifecycle.Lifecycle
+import androidx.lifecycle.ViewModelProvider
+import androidx.lifecycle.lifecycleScope
+import androidx.lifecycle.repeatOnLifecycle
+import dagger.hilt.android.AndroidEntryPoint
+import kotlinx.coroutines.launch
+import net.gini.android.capture.screen.R
+import net.gini.android.capture.screen.databinding.ActivityConfigurationBinding
+import net.gini.android.capture.screen.ui.MainActivity.Companion.CONFIGURATION_BUNDLE
+import net.gini.android.capture.screen.ui.data.Configuration
+
+
+@AndroidEntryPoint
+class ConfigurationActivity : AppCompatActivity(R.layout.activity_configuration) {
+
+    private lateinit var binding: ActivityConfigurationBinding
+    private lateinit var configuration: Configuration
+    private lateinit var configurationViewModel: ConfigurationViewModel
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        binding = ActivityConfigurationBinding.inflate(layoutInflater)
+        setContentView(binding.root)
+        configurationViewModel = ViewModelProvider(this)[ConfigurationViewModel::class.java]
+
+        configuration = intent.getParcelableExtra(CONFIGURATION_BUNDLE) ?: Configuration()
+        configurationViewModel.setConfiguration(configuration)
+
+        lifecycleScope.launch {
+            repeatOnLifecycle(Lifecycle.State.STARTED) {
+                configurationViewModel.configuration.collect {
+                    configuration = it
+                }
+            }
+        }
+
+        updateUIWithConfigurationObject()
+        setConfigurationFeatures()
+        binding.buttonCloseConfiguration.setOnClickListener {
+            returnToMainActivity()
+        }
+    }
+
+    override fun onBackPressed() {
+        returnToMainActivity()
+    }
+
+    private fun returnToMainActivity() {
+        val returnIntent = Intent().putExtra(CONFIGURATION_BUNDLE, configuration)
+        setResult(Activity.RESULT_OK, returnIntent)
+        finish()
+    }
+
+    private fun updateUIWithConfigurationObject() {
```

Comment:
[Request] This function should receive the configuration as a parameter and be invoked in the flow collector with the latest configuration.